src/components/DrinkDetails/DrinkDetails.js

```diff
@@ -1,97 +1,95 @@
-        strDrink: "",
-        idDrink: "",
-        strDrinkThumb: "",
-    };
+  state = {
+    ingredients: [],
+    strInstructions: '',
+    strGlass: '',
+    strDrink: '',
+    idDrink: '',
+    strDrinkThumb: '',
+  };
 
-    componentDidMount() {
-        console.log("[componentDidMount]", this.props.match.params.id);
-        if (!this.state.idDrink.length > 0) {
-            this.setState({ idDrink: this.props.match.params.id });
-            this.fetchDrinkData();
-        }
+  componentDidMount() {
+    console.log('[componentDidMount]', this.props.match.params.id);
+    if (!this.state.idDrink.length > 0) {
+      this.setState({ idDrink: this.props.match.params.id });
+      this.fetchDrinkData();
     }
+  }
 
-    fetchDrinkData = async () => {
-        let data = await axios.get(
-            "/v1/1/lookup.php?i=" + this.props.match.params.id
-        );
-        console.log("[fetchDrinkData]", data.data.drinks[0]);
-        data = data.data.drinks[0];
-
-        let ingredients = [];
-        for (let i = 1; i <= 15; i++) {
-            if (
-                data[`strIngredient${i}`] !== null &&
-                data[`strIngredient${i}`] !== ""
-            ) {
-                ingredients.push({
-                    measure: data[`strMeasure${i}`],
-                    ingredient: data[`strIngredient${i}`],
-                });
-            }
-        }
+  fetchDrinkData = async () => {
+    let data = await axios.get(
+      '/v1/1/lookup.php?i=' + this.props.match.params.id,
+    );
+    console.log('[fetchDrinkData]', data.data.drinks[0]);
+    data = data.data.drinks[0];
 
-        this.setState({
-            ingredients: ingredients,
-            strInstructions: data.strInstructions,
-            strDrink: data.strDrink,
-            strGlass: data.strGlass,
-            strDrinkThumb: data.strDrinkThumb,
+    let ingredients = [];
+    for (let i = 1; i <= 15; i++) {
+      if (
+        data[`strIngredient${i}`] !== null &&
+        data[`strIngredient${i}`] !== ''
+      ) {
+        ingredients.push({
+          measure: data[`strMeasure${i}`],
+          ingredient: data[`strIngredient${i}`],
         });
-        console.log(this.state);
-    };
+      }
+    }
 
-    render() {
-        let drinkDetails = <h2>Loading, one sec!</h2>;
-        if (this.state.ingredients.length > 0) {
-            let ingredientList = this.state.ingredients.map((ing) => {
-                return (
-                    <li>
-                        <p className={classes.ListItem}>
-                            <strong>{`${ing.measure}`}</strong>
-                            {` - ${ing.ingredient}`}
-                        </p>
-                    </li>
-                );
-            });
+    this.setState({
+      ingredients: ingredients,
+      strInstructions: data.strInstructions,
+      strDrink: data.strDrink,
+      strGlass: data.strGlass,
+      strDrinkThumb: data.strDrinkThumb,
+    });
+    console.log(this.state);
```

Comment:
Again I guess the `console.log`s are just kept in for debugging just something not to get into the habit of letting them go into master branch

Reply:
Thanks!